app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    cmd, origin_file, path = command.split()
+    if cmd == "mv":
+        if "/" in path:
+            part_of_path = path.split("/")
+            part_of_path.pop()
+            new_path = "/".join(part_of_path)
```

Comment:
Do not use "/" to check if directory is present, it won't work on every OS. Use os.path.split and os.path.join